script.js

```diff
@@ -1 +1,341 @@
-console.log('yay!');
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
```

Comment:
What's the reason for disabling these? They should be helpful in detecting unused code and `let`s that should be `const`s!

Reply:
the main reason I disable prefer const cause whenever I save, Eslint will automatically change the let variables that I need to const.

Reply:
Noted. For file-wide changes, consider adding to .eslintrc.js instead of here, and adding a comment to explain why :)

Personally I think this rule is good because it helps us catch `let` variables that are not reassigned and should be `const` instead.